disk.go

```diff
@@ -13,31 +13,48 @@ const (
 	DqdChunks
 )
 
+var idCounter int
 //==================================================================
```

Comment:
It might be hard to get it globally unique I think. monotonously increasing value was the simplest to get a unique value.  I shall try and see if I can get a function of node id which would be globally unique.